### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by SabMueller**
Hi everyone,

I managed to finish the following: 
- Contact.js where the user can get contact information for the animal they want to adopt
- added local stoarge for the selected animal / organization

Thanks for your review ✨

### CODE REVIEW

File: client/src/pages/Contact.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+const AnimalProfilePicture = styled.img`
+  max-height: 27rem;
+  max-width: 90vw;
+`;
+
+const AnimalName = styled.h2`
+  color: var(--secondary);
+  font-family: var(--ff-cursive);
+  font-size: 2rem;
+
```
Comment (OliverNeuhaeusler) [inline client/src/pages/Contact.js]: why this empty space?

File: client/src/pages/Contact.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  color: var(--black);
+  border-radius: 2rem;
+  filter: drop-shadow(0 0 0.2rem var(--black));
+  width: 90vw;
+  padding: 1rem;
+  margin: 2rem 0;
+  text-align: center;
+  display: flex;
+  flex-flow: column wrap;
+  gap: 1rem;
```
Comment (OliverNeuhaeusler) [inline client/src/pages/Contact.js]: maybe you can sort it alphabetically

**Review by OliverNeuhaeusler — APPROVED** [review]
Looks good to me!!

File: client/src/pages/Contact.js (hunk 1 above)
Comment (SabMueller) [reply client/src/pages/Contact.js]: I have no idea :D I removed it, thank you!

File: client/src/pages/Contact.js (hunk 2 above)
Comment (SabMueller) [reply client/src/pages/Contact.js]: good point!

**Pull request merged**